src/network/serverpackethandler.cpp

```diff
@@ -834,11 +836,12 @@ void Server::handleCommand_Password(NetworkPacket* pkt)
 	std::string newpwd;
 
 	// Deny for clients using the new protocol
-	RemoteClient* client = getClient(pkt->getPeerId(), CS_Created);
+	session_t peer_id = pkt->getPeerId();
+	RemoteClient* client = getClient(peer_id, CS_Created);
 	if (client->net_proto_version >= 25) {
-		infostream << "Server::handleCommand_Password(): Denying change: "
-			<< " Client protocol version for peer_id=" << pkt->getPeerId()
-			<< " too new!" << std::endl;
+		infostream << "Server::handleCommand_Password(): Denying change: " <<
+			"Client protocol version for peer_id=" << peer_id << " too new!" <<
+			std::endl;
```

Comment:
This function is dead at least since 0.5.0-dev. Feel free to purge it.

Reply:
done